mkdocs.yml

```diff
@@ -1,204 +1,237 @@
 site_name: EdgeX Foundry Documentation
-docs_dir: ./docs_src
-site_dir: ./docs/3.1
 site_description: 'Documentation for use of EdgeX Foundry'
-site_author: 'Michael Johanson'
-site_url: 'https://docs.edgexfoundry.org/3.1'
+site_author: 'Jim White'
+site_url: 'https://github.com/edgexfoundry/edgex-docs'
 repo_url: 'https://github.com/edgexfoundry/edgex-go'
 repo_name: 'edgex/edgex-go'
-copyright: 'Copyright &copy; 2022 EdgeX Foundry'
+copyright: 'Copyright &copy; 2023 EdgeX Foundry'
+docs_dir: docs_src
+site_dir: docs/3.1
 strict: false   # mkdocs-materials will prevent running on 0.0.0.0 otherwise (or have custom image)
+extra:
+  version:
+    provider: mike
```

Comment:
Move the version and provider to the extra section on line 251

Reply:
per meeting with @ernestojeda today - moved